lib/flipper/adapters/redis.rb

```diff
@@ -28,13 +28,21 @@ def features
 
       # Public: Adds a feature to the set of known features.
       def add(feature)
-        @client.sadd FeaturesKey, feature.key
+        if redis_sadd_returns_boolean?
+          @client.sadd? FeaturesKey, feature.key
+        else
+          @client.sadd FeaturesKey, feature.key
+        end
         true
       end
 
       # Public: Removes a feature from the set of known features.
       def remove(feature)
-        @client.srem FeaturesKey, feature.key
+        if redis_sadd_returns_boolean?
+          @client.srem? FeaturesKey, feature.key
```

Comment:
looking at https://github.com/mperham/sidekiq/issues 5484 I guess we could have 

```diff
-          @client.srem FeaturesKey, feature.key
+          @client.srem FeaturesKey, [feature.key]
```

I'm no redis expert, though